specs/MultiProfile.md

```diff
@@ -461,6 +576,44 @@ interface ICoreWebView2StagingProfileDeletedEventHandler: IUnknown {
 }
 ```
```

Comment:
the new interfaces aren't rendering correctly for me in rich view.  I think we meant to add these before the closing idl tag

Reply:
Thanks for reminding, updated the place of closing tag.